### PR TITLE
Dutch translation update

### PR DESCRIPTION
**Pull request by BasO**
Thorough reflection on an already great translation, with many small tweaks, mostly about further Dutchifying sentences, applying correct space usage within compound words, and improving text accessibility in general.

- [ ] This change has been discussed with the author.
- [ ] It implements / fixes issue #.

---

**Description:**
No new strings, just improvements to, and fine-tunings of the current strings by @dionvl and @ErikMinekus. Beyond my education in writing, other things taken into account were conventions in the Dutch macOS translation, and many cross-comparisons between IINA's english text, the Dutch strings and the excellent German translation by @shilch and @shorshe.

### CODE REVIEW

File: iina/nl.lproj/QuickSettingViewController.strings [hunk 1]
```diff
@@ -1,397 +1,134 @@
...
 "6LO-6y-IsA.title" = "Kleur:";
-
-/* Class = "NSMenuItem"; title = "5"; ObjectID = "7su-au-nwq"; */
 "7su-au-nwq.title" = "5";
-
-/* Class = "NSButtonCell"; title = "Load External Audio Track..."; ObjectID = "8Ax-5X-osl"; */
-"8Ax-5X-osl.title" = "Laad extern audio spoor...";
-
-/* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "9ss-EF-bby"; */
+"8Ax-5X-osl.title" = "Laad extern audiospoor...";
```
Comment (ErikMinekus) [inline iina/nl.lproj/QuickSettingViewController.strings]: Ellipsis

File: iina/nl.lproj/InspectorWindowController.strings [hunk 2]
```diff
@@ -223,16 +223,16 @@
 "jMN-aN-DqI.title" = "Label";
 
 /* Class = "NSTextFieldCell"; title = "Bit Rate:"; ObjectID = "jyV-Pd-UvH"; */
-"jyV-Pd-UvH.title" = "Bitrate:";
+"jyV-Pd-UvH.title" = "Bitsnelheid:";
 
 /* Class = "NSTextFieldCell"; title = "Sample Rate:"; ObjectID = "llF-4b-Z8b"; */
-"llF-4b-Z8b.title" = "Sample rate:";
+"llF-4b-Z8b.title" = "Sampling rate:";
```
[the remark this answers is not in the document]
Comment (BasO) [reply iina/nl.lproj/InspectorWindowController.strings]: Well found.

File: iina/nl.lproj/KeyBinding.strings [hunk 3]
```diff
@@ -18,49 +18,49 @@
...
 "cmd.quit" = "Afsluiten";
-"cmd.quit-watch-later" = "Afsluiten (onthoud voortgang)";
-"cmd.revert-seek" = "Door- of terugspoelen terugdraaien";
+"cmd.quit-watch-later" = "Afsluiten (bewaar voortgang)";
+"cmd.revert-seek" = "Herstel door- of terugspoelen";
 "cmd.run" = "Voer uit";
 "cmd.screenshot" = "Schermafbeelding";
 "cmd.screenshot-to-file" = "Schermafbeelding naar bestand";
-"cmd.seek" = "Door- of terugspoelen";
+"cmd.seek" = "Spoel door of terug";
```
[the remark this answers is not in the document]
Comment (BasO) [reply iina/nl.lproj/KeyBinding.strings]: Yeah, I changed everything for `spoel vooruit` (following Apple site wide - [example](https://support.apple.com/nl-nl/HT205305)) and must've missed this. Sharp eye.

File: iina/nl.lproj/MiniPlayerWindowController.strings [hunk 4]
```diff
@@ -31,4 +31,5 @@
 "aot-6J-Jp7.title" = "50";
 
 /* Class = "NSButton"; ibShadowedToolTip = "Close Mini Player"; ObjectID = "hZH-NM-Lst"; */
-"hZH-NM-Lst.ibShadowedToolTip" = "Sluit muziek modus";
+"hZH-NM-Lst.ibShadowedToolTip" = "Sluit minispeler";
```
[the remark this answers is not in the document]
Comment (BasO) [reply iina/nl.lproj/MiniPlayerWindowController.strings]: Looking at the original and German translation, and observing IINA's behavior, it seems that the music mode window with album art collapsed is called the mini player (e.g. when you hover on the close button). I don't think changing the name of music mode depending on showing album art is particularly sensible, but it appears that's what's going on in the broader picture.

File: iina/nl.lproj/KeyBinding.strings [hunk 5]
```diff
@@ -18,49 +18,49 @@
...
-"iina.toggle-flip" = "Omdraaien in- of uitschakelen";
-"iina.toggle-mirror" = "Spiegelen in- of uitschakelen";
-"iina.toggle-music-mode" = "Muziek modus in- of uitschakelen";
-"iina.toggle-pip" = "Beeld-in-beeld in- of uitschakelen";
-"iina.video-panel" = "Video paneel";
+"iina.sub-panel" = "Ondertitelingspaneel";
+"iina.toggle-flip" = "Schakel verticaal spiegelen in of uit";
+"iina.toggle-mirror" = "Schakel horizontaal spiegelen in of uit";
+"iina.toggle-music-mode" = "Schakel muziekmodus in of uit";
+"iina.toggle-pip" = "Schakel beeld in beeld in/uit";
```
[the remark this answers is not in the document]
Comment (BasO) [reply iina/nl.lproj/KeyBinding.strings]: Toen ik gisteren naar Apple's NL conventie keek zag ik alleen voorbeelden zonder streepje, dus ik nam aan dat het als een soort naam gebruikt wordt voor de systeemfunctie zoals uitgewerkt in iOS en macOS, en de spellingsregels dus niet noodzakelijk op hoeven te gaan. Maar nu ik verder kijk zie ik ook voorbeelden waar het met streepje geschreven wordt, dus dan ben ik inderdaad ook voor een streepje.

[the remark this answers is not in the document]
Comment (BasO) [reply iina/nl.lproj/KeyBinding.strings]: Vooralsnog ingezet op `Beeld-in-beeld`

File: iina/nl.lproj/MiniPlayerWindowController.strings (hunk 4 above)
[the remark this answers is not in the document]
Comment (ErikMinekus) [reply iina/nl.lproj/MiniPlayerWindowController.strings]: I believe it was always called mini player internally, but music mode to the user. See the original PR #963. Unfortunately some of the base phrases still say "mini player", which makes it confusing.

@lhc70000 can you explain if the user should see "mini player" or "music mode"?

**Comment by BasO** [comment]
Good review. Working on this translation, I realised that IINA's code doesn't correctly localise the format of Doubles - it shows a dot instead of a comma for parameter values such as `%.2fs`. For consistency with non-auto-generated numbers in our translated strings, the Doubles should output with a comma for Dutch users. Is it worth it to make an issue around this or is there an easy fix? I'm not sure how to go about changing this in the code myself.

**Comment by alejx** [comment]
Please revert the changes in the .xib files.

**Comment by BasO** [comment]
Wasn't sure how, is this okay? I reverted all commits then added all .strings changes in a single commit

**Comment by BasO** [comment]
For the decimal separator issue, I created https://github.com/iina/iina/issues/2597

**Comment by alejx** [comment]
@BasO Thanks for your great work. 
> Wasn't sure how to do that in a clean process, is this okay?

It's okay, as long as there are no changes to files other than string files in the diff. I'll do a rebase before merging so it won't be so messy then.

**Pull request merged**